wormhole-connect/src/views/v2/Bridge/ReviewTransaction/index.tsx

```diff
@@ -210,6 +211,9 @@ const ReviewTransaction = (props: Props) => {
         throw new Error("Can't find txid in receipt");
       }
 
+      // Set the start time of the transaction
+      dispatch(setTimestamp(Date.now()));
```

Comment:
Note: This way we will have a start timestamp for a new transaction, but still need a solution to get the start time for "Resume transaction".
Cc: @artursapek

Reply:
since we can only resume txs that are finalized on the source chain & attested, the ETA for a resume transaction is just the destination chain finality time. you can actually just get this using `finality` in the SDK: https://github.com/wormhole-foundation/wormhole-sdk-ts/blob/main/connect/src/protocols/tokenBridge/tokenTransfer.ts#L720

Reply:
since we can only resume transfers that have been finalized on the source chain & attested, there's no real ETA to show for them. as soon as the destination transaction goes through, the transfer is complete. so I think there's no need for an ETA for those.

Reply:
@artursapek What if the user took note of the tx hash and came back a few mins later to resume it? It'd still be in source finality I believe.

Reply:
I think this is good for internal release. I've opened an item to revisit this:
https://www.notion.so/wormholelabs/UI-Feature-Do-we-need-a-transaction-creation-date-in-the-SDK-a8969f1344dc4583bd50eddb62009694?pvs=4